liquidprompt

```diff
@@ -1479,21 +1481,20 @@ _lp_load_color()
 
     (( LP_ENABLE_LOAD )) || return
 
-    local lp_cpu_load
-    # Get value (OS-specific) into lp_cpu_load
-    _lp_cpu_load
+    local lp_loadavg centiload
+    # Get value (OS-specific) into lp_loadavg
+    _lp_loadavg
 
-    lp_cpu_load=${lp_cpu_load/./}   # Remove '.'
-    lp_cpu_load=${lp_cpu_load#0}    # Remove leading '0'
-    lp_cpu_load=${lp_cpu_load#0}    # Remove leading '0', again (ex: 0.09)
-    local -i load=${lp_cpu_load:-0}/$_lp_CPUNUM
+    centiload=${lp_loadavg/./}   # Remove '.'
+    centiload=${centiload#0}     # Remove leading '0'
+    centiload=${centiload#0}     # Remove leading '0', again (ex: 0.09)
+    local -i centiload_per_cpu=${centiload:-0}/$_lp_CPUNUM
 
-    if (( load > LP_LOAD_THRESHOLD )); then
-        local ret="$(_lp_color_map $load)${LP_MARK_LOAD}"
+    if (( centiload_per_cpu > LP_LOAD_THRESHOLD )); then
+        local -i ceiling=$LP_LOAD_THRESHOLD*20
+        local ret="$(_lp_color_map $centiload_per_cpu $ceiling)${LP_MARK_LOAD}"
```

Comment:
The problem with load average (as you have pointed out) is that there is no upper bound. You're right, this feature is quite broken. But, I'm not convinced that setting a random `20` is better than leaving it at `1`.